lib/components/Tray.js

```diff
@@ -11,7 +11,8 @@ export default React.createClass({
     onBlur: React.PropTypes.func,
     closeTimeoutMS: React.PropTypes.number,
     closeOnBlur: React.PropTypes.bool,
-    maintainFocus: React.PropTypes.bool
+    maintainFocus: React.PropTypes.bool,
+    focusAfterOpen: React.PropTypes.string
```

Comment:
I think it'd be better to call this `elementToFocus` and have it be React.PropTypes.element. It's more descriptive and would cause less confusion since there is already a focusAfterRender prop.